HHS/OPRE-OPS
# Ops 2504 concurrent fake users

## What changed

Disable `UserSession` check for `FakeAuth`

N.B. This effectively causes the `UserSession` to be "shared" with other computers/browsers logged in as the same "Fake" user.

## Issue

https://github.com/HHS/OPRE-OPS/issues/2504

## How to test

1. Log in as the Admin User
2. Find the active `UserSession` record in the DB and change the IP address
3. Log in as the Admin User using a different browser, e.g. Edge or Firefox

Previous to this change, step 3 would have prevented you from logging in using the second browser and logged you out in the first.  The `UserSession` record in the DB would be set to `is_active=false`.

With this change, the ip address isn't checked for fake users and so theoretically you should be able to log in using multiple browsers/computers.

## Definition of Done Checklist
